packages/react/react-query/package.json

```diff
@@ -11,7 +15,7 @@
   },
   "main": "./src/index.ts",
   "scripts": {
-    "build": "rm -rf dist esm && tsc --declaration --emitDeclarationOnly --declarationDir dist && rollup -c rollup.config.js",
+    "build": "rm -rf dist esm && tsc --declaration --emitDeclarationOnly --declarationDir dist && rollup -c rollup.config.js && pnpify tsd",
```

Comment:
### Check it is fine please
I chained ```pnpify tsd``` after original build
It will make error in process of build if there is error in tsd code.

Is it fine for you?